servicemesh-install/topics/install-multi-tenant.adoc

```diff
@@ -0,0 +1,77 @@
+[[multi-tenant-install]]
+= Multi-tenant {ProductName} installation
+
+The {ProductName} operator provides support for multi-tenant control plane installations. A multi-tenant control plane is configured so that only specified namespaces can be joined into its {ProductShortName}, isolating the mesh from other installations.
+
+[NOTE]
+====
+You cannot use multi-tenant control plane installations in conjunction with a cluster-wide control plane installation. {ProductName} installations must either be multi-tenant or single, cluster-wide installations.
+====
+
+[WARNING]
+====
+Automatic route creation is currently incompatible with multi-tenant {ProductShortName} installations. Ensure that it is disabled, by setting `ior_enabled` to `false` in your `ServiceMeshControlPlane` if you plan to attempt a multi-tenant installation.
+====
+
+== Known issues
+
+* `MeshPolicy` is still a cluster-scoped resource and applies to all control planes installed in OpenShift. This can prevent the installation of multiple control planes or cause unknown behavior if one control plane is deleted
+* The Jaeger agent runs as a `DaemonSet`, therefore `tracing` may only be enabled for a single `ServiceMeshControlPlane` instance.
+* If you delete the project that contains the control plane before you delete the `ServiceMeshControlPlane` resource, some parts of the installation may not be removed:
+** Service accounts added to the `SecurityContextConstraints` may not be removed
+** `OAuthClient` resources associated with Kiali may not be removed, or its list of redirectURIs may not be accurate
+
+== Comparison of multi-tenant and cluster-wide installations
+
+The main difference between a multi-tenant installation and a cluster-wide installation is the scope of privileges used by the control plane deployments, for example, galley and pilot. The components no longer use cluster-scoped Role Based Access Control (RBAC) `ClusterRoleBinding`, but rely on namespace-scoped RBAC `RoleBinding`. Every namespace in the `members` list will have a `RoleBinding` for each service account associated with a control plane deployment and each control plane deployment will only watch those member namespaces. Each member namespace has a `maistra.io/member-of` label added to it, where the `member-of` value is the namespace containing the control plane installation.
```

Comment:
galley and pilot.  Should these be capitalized?  Istio capitalizes the various architecture components, we should probably follow that lead.

Reply:
Fixed!